ComputerInterface/CustomKeyboardKey.cs

```diff
@@ -87,9 +87,28 @@ public void Init(CustomComputer computer, EKeyboardKey key, Text keyboardText, s
         public void Init(CustomComputer computer, EKeyboardKey key, Text keyboardText, string text, Color buttonColor)
         {
             Init(computer, key, keyboardText, text);
+
+            if (_material == null)
+            {
+                _originalColor = buttonColor;
+
+                var baseRenderer = GetComponent<Renderer>();
+                if (baseRenderer.material == null)
+                {
+                    _material = new Material(Shader.Find("Legacy Shaders/Diffuse"))
+                    {
+                        color = buttonColor
+                    };
+                    goto PrepareColour;
+                }
+                baseRenderer.material.color = buttonColor;
+                goto PrepareColour;
+            }
+
             _material.color = buttonColor;
             _originalColor = buttonColor;
 
+            PrepareColour:
```

Comment:
This logic doesn't make sense, why is `_originalColor` always set to the new color? Also, GOTO considered harmfull, you should almost never use it